files/jcl/ZWENOSEC.jcl

```diff
@@ -203,6 +203,15 @@
   LISTGRP  &ADMINGRP. OMVS
   DELGROUP &ADMINGRP.
 
+/* REMOVE ZOWE RESOURCE CLASS ...................................... */
+
+/* remove profile in ZOWE class                                      */
+  RDELETE ZOWE APIML.SERVICES
+/* remove ZOWE class                                                 */
+  RDELETE CDT ZOWE
```

Comment:
It might be already defined on the system before. In this case, we should remove it.